README.md

```diff
@@ -259,8 +259,7 @@ pub fn hello_from_python() -> FnResult<String> {
 ### Testing it out
 
 We can't really test this from the Extism CLI as something must provide the implementation. So let's
-write out the python side here. You should check the docs for the language you're using on the host side
-for how to implement host functions.
+write out the Python side here. Check out the [docs for the host SDKs](https://extism.org/docs/category/integrate-into-your-codebase) to implement a host function in a language of your choice.
```

Comment:
This page will be going away I think. Might be better to link to the Host SDK concept doc?